CITATION.cff

```diff
@@ -0,0 +1,46 @@
+# YAML 1.2
+# Metadata for citation of this software according to the CFF format (https://citation-file-format.github.io/)
+cff-version: 1.1.0
+message: If you use this software, please cite it using these metadata.
+title: "MatterMiners/tardis: 0.7.1"
```

Comment:
This is kind of chicken and egg problem. I have to add the "link" to release 0.7.1, since this is the current one published on Zenodo and have to update it to version 0.8.0 once it is released.

Reply:
Can't you just write 0.8.0? Because in time it is released, it will be valid?

Reply:
Yes, but I do not know the doi of that version in advance.

Reply:
You could go with the generic doi for tardis itself as well. That's `https://doi.org/10.5281/zenodo.2240605`.

But I see no problem having the old version here until the release is made and then updating the info afterwards. It doesn't seem to get packaged, so it's not strictly attached to a version anyway - only the repo.